js/util.js

```diff
@@ -111,4 +111,15 @@ export const reduceCount = (count, digits = 1) => {
 
   // Do not reduce if we are below 1000
   return divisor ? (count / divisor).toFixed(digits) + indicator : count
-}
\ No newline at end of file
+}
+
+export const SortingOptions = [
+  { value: 'new', text: 'New' },
+  { value: 'top', text: 'Top' },
+  { value: 'old', text: 'Old' },
+  { value: 'controversial', text: 'Controversial' },
+  { value: 'random', text: 'Random' },
+  { value: 'qa', text: 'QA' },
+  { value: 'live', text: 'Live' },
```

Comment:
Reading on this mode I think we should probably comment it out for now, only applies to certain special threads and is meant for streaming comments, which we don't currently support.. Not sure if all the other ones can actually be used with any thread.

Reply:
Commented out for now.